src/test/java/uk/gov/companieshouse/web/accounts/service/smallfull/impl/SmallFullServiceImplTests.java

```diff
@@ -0,0 +1,146 @@
+import static org.mockito.Mockito.verify;
+import static org.mockito.Mockito.when;
+
+import org.junit.jupiter.api.BeforeEach;
+import org.junit.jupiter.api.DisplayName;
+import org.junit.jupiter.api.Test;
+import org.junit.jupiter.api.TestInstance;
+import org.junit.jupiter.api.extension.ExtendWith;
+import org.mockito.InjectMocks;
+import org.mockito.Mock;
+import org.mockito.junit.jupiter.MockitoExtension;
+import uk.gov.companieshouse.api.ApiClient;
+import uk.gov.companieshouse.api.error.ApiErrorResponseException;
+import uk.gov.companieshouse.api.handler.exception.URIValidationException;
+import uk.gov.companieshouse.api.handler.smallfull.SmallFullResourceHandler;
+import uk.gov.companieshouse.api.handler.smallfull.request.SmallFullCreate;
+import uk.gov.companieshouse.api.handler.smallfull.request.SmallFullGet;
+import uk.gov.companieshouse.api.model.accounts.smallfull.SmallFullApi;
+import uk.gov.companieshouse.web.accounts.api.ApiClientService;
+import uk.gov.companieshouse.web.accounts.exception.ServiceException;
+import uk.gov.companieshouse.web.accounts.service.smallfull.SmallFullService;
+
+@ExtendWith(MockitoExtension.class)
+@TestInstance(TestInstance.Lifecycle.PER_CLASS)
+public class SmallFullServiceImplTests {
+
+    @Mock
+    private ApiClient apiClient;
+
+    @Mock
+    private ApiClientService apiClientService;
+
+    @Mock
+    private SmallFullResourceHandler smallFullResourceHandler;
+
+    @Mock
+    private SmallFullCreate smallFullCreate;
+
+    @Mock
+    private SmallFullGet smallFullGet;
+
+    @InjectMocks
+    private SmallFullService smallFullService = new SmallFullServiceImpl();
+
+    private static final String TRANSACTION_ID = "transactionId";
+
+    private static final String COMPANY_ACCOUNTS_ID = "companyAccountsId";
+
+    @BeforeEach
+    private void init() {
+
+        when(apiClientService.getApiClient()).thenReturn(apiClient);
+
+        when(apiClient.smallFull()).thenReturn(smallFullResourceHandler);
+    }
+
+    @Test
+    @DisplayName("Create Small Full Accounts - Success Path")
+    void createSmallFullSuccess() throws ServiceException, ApiErrorResponseException, URIValidationException {
+
+        when(smallFullResourceHandler.create(anyString(), any(SmallFullApi.class)))
+                .thenReturn(smallFullCreate);
+
+        when(smallFullCreate.execute()).thenReturn(new SmallFullApi());
+
+        smallFullService.createSmallFullAccounts(TRANSACTION_ID, COMPANY_ACCOUNTS_ID);
+
+        verify(smallFullCreate, times(1)).execute();
+    }
+
+    @Test
+    @DisplayName("Create Small Full Accounts - Throws ApiErrorResponseException")
+    void createSmallFullApiErrorResponseExceptionThrown() throws ApiErrorResponseException, URIValidationException {
+
+        when(smallFullResourceHandler.create(anyString(), any(SmallFullApi.class)))
+                .thenReturn(smallFullCreate);
+
+        when(smallFullCreate.execute())
+                .thenThrow(ApiErrorResponseException.class);
```

Comment:
don't really think you need this `thenThrow` on the next line, it can be quite difficult to read when things are split too much.